### PR TITLE
The great hydroponics belt

### PR DESCRIPTION
**Pull request by SonicGotNuked**
**REASON**

I bloody hate how I have to throw all my crap into my backpack and I don't have a belt that can store my crap onto my belt slot. This aims to have a hydroponics belt be added via the lockers and bio-thingy.

Also, it'll add a belt that defines hydroponics, and has a few things added to it so it gives you a reason to use it instead of the regular utility belt. 

Also, I'm removing the chemical-grenade restriction to cleaning from the janitor belt. Now it can hold all grenades, same with this new belt.

**WORK**

- [x] Sprites
belt dmi

- [x] Code
- [x] Mapping/Actually spawning it in to xenobotany, and hydroponics.
- [x] Testing
_It's alive, it's working, and it didn't set my computer on fire_

**CONCLUSION** 
I have no clue what I'm doing.

### CODE REVIEW

**Comment by SonicGotNuked** [comment]
!wip

**Comment by SonicGotNuked** [comment]
!review

**Comment by Butterrobber202** [comment]
very sexy and useful, big approval.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -312,6 +312,43 @@
...
 
+/obj/item/weapon/storage/belt/hydro
+	name = "hydrobelt"
+	desc = "A utility belt to store and provide easy access to your floral utilities."
+	icon_state = "growbelt"
+	item_state = "growbelt"
+	storage_slots = 9
+	w_class = 3
+	max_w_class = 4
+//	max_storage_space = 16 *Can't decide, storage space could allow this to become the new medical belt meta. Leaving disabled to avoide green belt doctors
```
Comment (skull132) [inline code/game/objects/items/weapons/storage/belt.dm]: Do not commit dead code.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -312,6 +312,43 @@
...
+		/obj/item/weapon/reagent_containers/food/snacks/grown,
+		/obj/item/seeds,
+		/obj/item/weapon/grown, 
+		/obj/item/weapon/material/minihoe,
+		/obj/item/weapon/material/hatchet,
+		/obj/item/weapon/wirecutters,
+		/obj/item/weapon/reagent_containers/spray, //includes if you ever wish to get a spraybottle full of other chemicals, Like water
+		/obj/item/device/analyzer/plant_analyzer,
+		/obj/item/clothing/gloves/botanic_leather,
+		)
```
Comment (skull132) [inline code/game/objects/items/weapons/storage/belt.dm]:
```suggestion
		)

```

readability please.

**Comment by SonicGotNuked** [comment]
Changes that were requested has been made, removed the dead code + added a space for readability.

**Pull request merged**